### ISSUE TITLE
Rocket Chat Migration.

### ISSUE THREAD

**Issue by MoHD20** [RocketChat/Rocket.Chat.Ansible#77]
Could the migration process be implemented? My thoughts is that it's just migrating MongoDB right?

**Comment by xenithorb** [comment]
Can you describe in better detail what it is you're looking for?

**Comment by MoHD20** [comment]
I was thinking of migrating an already deployed Rocket chat server to a new server with more specs for example.

**Comment by xenithorb** [comment]
While technically possible, automating "from one server to another" seems a bit outside of the scope of this role. However, what does make some sense to support is restoring an already existing mongodb databse dump, after mongodb is installed by this role. I like it :+1:

**Comment by MoHD20** [comment]
But what is the use cases of restoring an already existing mongodb database dump to the same server? Things like automating backups & restoring?

**Comment by jlozadad** [comment]
yes, basically that. You can somewhat take this further add the `copy` module, send it to the other RC host and possibly restore there but, there is no mongodb module to restore like pgsql. So you will have to use `shell/command` module which can get tricky.

**Comment by nickbroon** [comment]
Having ansible tasks and variables to specify a database dump to restore would be great for deploying a new RocketChat server with data from an old manually installed system.
(And perhaps having ansible task that configure automatic database back dumps would also be great).
